sdk/identity/Azure.Identity/README.md

```diff
@@ -263,6 +263,11 @@ Not all credentials require this configuration. Credentials which authenticate t
 |`AZURE_USERNAME`|a username (usually an email address)
 |`AZURE_PASSWORD`|that user's password
 
+#### Managed identity
+|Variable name|Value
+|-|-
+|`AZURE_CLIENT_ID`|Used as the default value for `ManagedIdentityClientId` in `DefaultAzureCredentialOptions`
```

Comment:
I'm a little confused, though. By default it's not set, so what gets used then? It works without it, or is it that wherever MSI is supposed, `AZURE_CLIENT_ID` is defined?

Reply:
nothing is used if the environment is not defined. It is only required to set this if you intend to use a user-defined identity.